apiary.apib

```diff
@@ -3723,13 +3723,32 @@ Finalize a draft work order. This will also generate the work order PDF.
 
 Send a finalized work order to the customer.
 
-+ Request (application/json;charset=utf-8)
++ Request (application/json)
 
     + Attributes (object)
         + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
 
 + Response 204
 
+### workOrders.download [POST /workOrders.download]
+
+Download a work order in a specific format.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
+            + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
```

Comment:
Should be `expires_at` actually. Seems like we made this mistake for downloading invoices as well.

Reply:
We'll need to discuss how to keep this consistent on all the `.download` endpoints.